### PR TITLE
Deploy falcon/injector to kubernetes as batch/Job

### PR DESCRIPTION
**Pull request by isimluk**

### CODE REVIEW

File: apis/falcon/v1alpha1/falconconfig_types.go [hunk 1]
```diff
@@ -16,6 +16,9 @@ type FalconAPI struct {
 	ClientId string `json:"client_id"`
 	// Falcon OAuth2 API Client Secret
 	ClientSecret string `json:"client_secret"`
+	// Falcon Customer ID (CID)
+	// +kubebuilder:validation:Pattern="[0-9a-zA-Z]+-[0-9a-zA-Z]{2}"
```
Comment (redhatrises) [inline apis/falcon/v1alpha1/falconconfig_types.go:20]: Can you make the regex `^[0-9a-fA-F]{32}-[0-9a-fA-F]{2}$` or `[0-9a-fA-F]{32}-[0-9a-fA-F]{2}`

File: bundle/manifests/falcon.crowdstrike.com_falconconfigs.yaml [hunk 2]
```diff
@@ -33,6 +33,10 @@ spec:
               falcon_api:
                 description: FalconAPI configures connection from your local Falcon operator to CrowdStrike Falcon platform.
                 properties:
+                  cid:
+                    description: Falcon Customer ID (CID)
+                    pattern: '[0-9a-zA-Z]+-[0-9a-zA-Z]{2}'
```
Comment (redhatrises) [inline bundle/manifests/falcon.crowdstrike.com_falconconfigs.yaml:38]: Can you make the regex `^[0-9a-fA-F]{32}-[0-9a-fA-F]{2}$` or `[0-9a-fA-F]{32}-[0-9a-fA-F]{2}`

File: config/crd/bases/falcon.crowdstrike.com_falconconfigs.yaml [hunk 3]
```diff
@@ -40,6 +40,10 @@ spec:
                 description: FalconAPI configures connection from your local Falcon
                   operator to CrowdStrike Falcon platform.
                 properties:
+                  cid:
+                    description: Falcon Customer ID (CID)
+                    pattern: '[0-9a-zA-Z]+-[0-9a-zA-Z]{2}'
```
Comment (redhatrises) [inline config/crd/bases/falcon.crowdstrike.com_falconconfigs.yaml:45]: Can you make the regex `^[0-9a-fA-F]{32}-[0-9a-fA-F]{2}$` or `[0-9a-fA-F]{32}-[0-9a-fA-F]{2}`

File: apis/falcon/v1alpha1/falconconfig_types.go (hunk 1 above)
Comment (isimluk) [reply apis/falcon/v1alpha1/falconconfig_types.go]: Yes, I can.

File: bundle/manifests/falcon.crowdstrike.com_falconconfigs.yaml (hunk 2 above)
Comment (isimluk) [reply bundle/manifests/falcon.crowdstrike.com_falconconfigs.yaml]: Yes, I can.

File: config/crd/bases/falcon.crowdstrike.com_falconconfigs.yaml (hunk 3 above)
Comment (isimluk) [reply config/crd/bases/falcon.crowdstrike.com_falconconfigs.yaml]: Yes, I can.

**Comment by redhatrises** [comment]
LGTM

**Pull request merged**